CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- First available seller being used instead of default seller
```

Comment:
I suggest making this phrase more clear to avoid misunderstanding, change the phrase order to something like that:
"Seller to be the default seller instead of the first available seller".

Reply:
I write this because i understood that should be used the opposite